### ISSUE TITLE
Question about voxel size and point cloud range

### ISSUE THREAD

**Issue by Len-Li** [TRAILab/CaDDN#56]
Hi, thanks for sharing your great work!
I want to enlarge voxel size for less memory usage. I notice that there are two factors affect 3d voxel shape: VOXEL_SIZE and POINT_CLOUD_RANGE. In `POINT_CLOUD_RANGE: [2, -30.08, -3.0, 46.8, 30.08, 1.0]`, the forward range is limited into (2,46.8), is it mean that the far away objects are ignored from training data? What's the intuition behind it?
Thanks!

**Comment by codyreading** [comment]
Thanks for the interest!

Yup, the forward range is limited to (2,46.8). I've done experiments increasing the range, which leads to no performance gain. Essentially, the detection performance past 46.8 meters is poor, so there is no point trying to detect objects in that range (especially with monocular methods). Adding additional range either requires a reduction of resolution or an increase in GPU memory.

**Comment by Len-Li** [comment]
I got it. Thanks for your reply.

By the way, have you read [Imvoxelnet](https://github.com/saic-vul/imvoxelnet)? I think Imvoxelnet is just a degeneration of CaDDN(CaDDN without depth distribution). While Imvoxelnet showing better performance in val-set. It doesn't make sense. How do you think about it?

**Comment by codyreading** [comment]
I haven't read it but I am familiar with the idea, as it is even similar to [Orthographic Feature Transform](https://arxiv.org/abs/1811.08188). It's hard to say concretely why ImVoxelBet shows better performance on the val-set, but one thing to look at is the 0.08m voxel size that ImVoxelNet uses compared to the 0.16m voxel size that CaDDN uses, where the higher resolution might be more important on val set, but the depth distribution estimation is more beneficial on test set. Additionally, it might be that the real benefit of the depth distribution estimation comes when training on the full train + val set as opposed to just the train set.
